### PR TITLE
fix(date-picker): fix the props(align)

### PR DESCRIPTION
**Pull request by 17biubiu**
Please make sure these boxes are checked before submitting your PR, thank you!

* [ √ ] Make sure you follow Element's contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [ √] Make sure you are merging your commits to `dev` branch.
* [ √] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/time-picker/src/time-picker-com/panel-time-pick.vue [hunk 1]
```diff
@@ -92,6 +93,12 @@ export default defineComponent({
       if (props.format.includes('a')) return 'a'
       return ''
     })
+    // watch
+    watch(() => props.visible, val => {
+      if (val) {
+        oldValue.value = props.parsedValue
+      }
+    })
```
Comment (JeremyWuuuuu) [inline packages/time-picker/src/time-picker-com/panel-time-pick.vue:101]: cc @zazzaz I think you need to take a look at this code.

Comment (17biubiu) [reply packages/time-picker/src/time-picker-com/panel-time-pick.vue:101]: 貌似好久不见他出来了

File: packages/time-picker/src/common/picker.vue [hunk 2]
```diff
@@ -177,6 +178,12 @@ const valueEquals = function(a, b) {
   return false
 }
 
+const PLACEMENT_MAP = {
+  left: 'bottom-start',
+  center: 'bottom',
+  right: 'bottom-end',
+}
+
 export default defineComponent({
```
Comment (iamkun) [inline packages/time-picker/src/common/picker.vue:187]: Seems this config is not valid in the new popperjs@core?

Comment (JeremyWuuuuu) [reply packages/time-picker/src/common/picker.vue:187]: image
[Popperjs/@core](https://popper.js.org/docs/v2/constructors/)
Seems legit.

Comment (17biubiu) [reply packages/time-picker/src/common/picker.vue:187]: pending？？

**Comment by kooriookami** [comment]
fixed in #2267

**Pull request closed without merging**